analyzer/analyzer.go

```diff
@@ -906,6 +904,12 @@ func (pd *ParsedData) parseBugReport(fnameA, contentsA, fnameB, contentsB string
 			warnings,
 			errs, summariesOutput.overflowMs > 0, true)
 
+		if supV {
+			sensorserviceOutput = <-sensorserviceCh
+			sensorserviceOutput = constructAppName(sensorserviceOutput, data.AppStats)
+			errs = append(errs, sensorserviceOutput.ParsingErrs...)
+		}
```

Comment:
Why move this block separately?

Reply:
I moved this block because for the sensorserviceOutput data, I called "constructAppName" function on it to organize the app names. And the data.AppStats variable can only be constructed after running line 901.

I can still keep it at the same place to generate the sensorserviceOutput data and just call the "constructAppName" here, but I thought that keeping all the new lines in a block may help when merging the code.

Reply:
Either way is fine!! Whatever you prefer!